Proiect-PAO/src/Date/Date.java

```diff
@@ -0,0 +1,106 @@
+package Date;
+
+import java.util.Scanner;
```

Comment:
why did you create a class for Date since we have java.time package with classes for working with Dates and Time - LocalDate, LocalTime, LocalDateTime, etc